src/components/utdragFraSykefravaeret/Sykmeldinger.tsx

```diff
@@ -153,9 +153,12 @@ const SykmeldingTittelbeskrivelse = ({ sykmelding }: UtvidbarTittelProps) => {
   );
 };
 
-export default function Sykmeldinger() {
+interface Props {
+  selectedOppfolgingstilfelle: OppfolgingstilfelleDTO | undefined;
```

Comment:
Kan denne vær `undefined`? Er det ikke alltid valgt et oppfølgingstilfelle? 😊

Reply:
Ja, eller, ettersom man alltids kan slå opp en person i modia som aldri har vært sykmeldt før (for whatever reason), så kan den være `undefined` her. Evt måtte vi håndtert det utenfor komponenten og tvunget den til å være definert på dette tidspunktet.